server/src/services/paygov.es6

```diff
@@ -23,125 +24,102 @@ paygov.createToken = (permitId) => {
     subject: 'christmas tree permit orders',
     audience: 'fs-trees-permit-api-users'
   };
-  return jwt.sign(
-    {
-      data: permitId
-    },
-    vcapConstants.PERMIT_SECRET,
-    claims
-  );
+  return new Promise((resolve, reject) => {
+    const token = jwt.sign(
+      {
+        data: permitId
+      },
+      vcapConstants.PERMIT_SECRET,
+      claims
+    );
+    if (token) {
+      resolve(token);
+    }
+    reject(new Error('Unable to generate token'));
+  }); 
 };
 
 /**
- * @function createSuccessUrl - create success url for paygov request
+ * @function returnUrl - create success url for paygov request
  * @param {string} forestAbbr - forest abbreviation
  * @param {string} permitId - permit id
+ * @param {Boolean} cancel - whether to include the cancel query
```

Comment:
I don't know the effect of adding the query parameter that results from `cancel === true` but consider changing the variable name `cancel` to provide more context as to its effect such as `isCancel`.

Reply:
great suggestion!